rental.rb

```diff
@@ -0,0 +1,13 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [ ] Hello, 👏 it's truly commendable that you've included Getters for the `@book and @person` 🏡 This is great because it enables you to effortlessly access the current state of the rentals. However, keep in mind that you might also need to modify or update this state. To make this process smooth and maintain data integrity, it's a wise practice to implement Setters for the `@book and @person` variables as well. 🛠️
- [ ] Setters act as your control center for making changes to the state of `@book and @person`. 🔄 They allow you to update, add, or remove book and person information as needed, ensuring that your data stays accurate and your book and person operations run seamlessly. So, combining Getters and Setters creates a robust toolkit for effectively managing your rental system! 📊🔧

```suggestion
 attr_accessor :date, :book, :person
```